README.md

```diff
@@ -419,6 +439,45 @@ module.exports = function (MyModel) {
 }
 ```
 
+## Custom Errors
+Generic errors respond with a 500, but sometime you want to have a better control over the error that is returned to the client, taking advantages of fields provided by JSONApi.
+
+**It is recommanded to extend the base Error constructor, before throwing errors, ex: BadRequestError**
```

Comment:
It is recommended that you extend the base Error constructor before throwing errors. Eg. BadRequestError